spec/services/statistic_service_spec.rb

```diff
@@ -23,7 +24,7 @@
     end
 
     it "calls the allocation reporting service" do
-      expect(AllocationReportingService).to receive(:call).with(recruitment_cycle_scope: recruitment_cycle)
+      expect(AllocationReportingService).to receive(:call).with(recruitment_cycle_scope: RecruitmentCycle.find_by(year: Settings.allocation_cycle_year))
```

Comment:
perhaps we can put this into a variable using the factories, for consistency

```suggestion
      expect(AllocationReportingService).to receive(:call).with(recruitment_cycle_scope: allocation_recruitment_cycle)
```

There is also a trait for this, which makes it easier that finding the RC with the settings

Reply:
There's a bit of subtelty here as `previous_recruitment_cycle` and `allocation_recruitment_cycle` are the same cycle and in this spec both names are relevant as only the AllocationReportingService would be for `allocation_recruitment_cycle` the rest would make more sense for `previous_recruitment_cycle`,  I've gone for a hybrid approach in the end.